From 9143429eac46639c1d7969957ba79e0f741a42d4 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 7 May 2021 21:57:24 -0400
Subject: [PATCH] lib: export vsscanf

The function was missing from exports, even though it loooks like the
intent of the implementation in sscanf.c was to have it exported.
---
 include/vsprintf.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/vsprintf.h b/include/vsprintf.h
index 2290083eba..201c9ed077 100644
--- a/include/vsprintf.h
+++ b/include/vsprintf.h
@@ -234,6 +234,14 @@ char *strmhz(char *buf, unsigned long hz);
  */
 void str_to_upper(const char *in, char *out, size_t len);
 
+/**
+ * vsscanf - Unformat a buffer into a list of arguments
+ * @buf:	input buffer
+ * @fmt:	format of buffer
+ * @args:	arguments
+ */
+int vsscanf(const char *inp, char const *fmt0, va_list ap);
+
 /**
  * sscanf - Unformat a buffer into a list of arguments
  * @buf:	input buffer
-- 
2.29.2

